x-pack/plugins/data_visualizer/public/application/index_data_visualizer/search_strategy/requests/get_document_stats.ts

```diff
@@ -189,13 +187,9 @@ export const getDocumentCountStats = async (
     const newProbability =
       (initialDefaultProbability * numDocs) / (numSampled - 2 * Math.sqrt(numSampled));
 
-    // If the number of docs sampled is indicative of query with < 10 million docs
+    // If the number of docs sampled is is too small
     // proceed to make a vanilla aggregation without any sampling
-    if (
-      numSampled === 0 ||
-      newProbability === Infinity ||
-      numSampled / initialDefaultProbability < 1e7
-    ) {
+    if (numSampled === 0 || newProbability === Infinity || numSampled < 5) {
```

Comment:
Why is `5` chosen here? It seems to me it should be much bigger. 

Also, `numSampled === 0` already implies `numSampled < 5`, so that equals `0` check is redundant.